### PR TITLE
Connect expenditures with actions model

### PR DESCRIPTION
**Pull request by jakubcolony**
https://github.com/JoinColony/colonyCDapp/pull/1969/

### CODE REVIEW

**Review by bassgeta — APPROVED** [review]
code looks good, works like it should, LGTM :ship:

**Review by iamsamgibbs — APPROVED** [review]
Tested with colonyCDapp#1969

**Review by iamsamgibbs — CHANGES_REQUESTED** [review]
Oh scratch that, this seems to have created issues with creating simple payments via the action sidebar (403 response).

File: src/handlers/actions/editDomain.ts [hunk 1]
```diff
@@ -1,18 +1,17 @@
...
 } from '~utils';
 
 export default async (event: ContractEvent): Promise<void> => {
-  const receipt = await provider.getTransactionReceipt(event.transactionHash);
-  const hasDomainAddedEvent = receipt.logs.some((log) =>
-    log.topics.includes(utils.id(ContractEventsSignatures.DomainAdded)),
+  const hasDomainAddedEvent = await transactionHasEvent(
+    event.transactionHash,
+    ContractEventsSignatures.DomainAdded,
   );
```
Comment (rdig) [inline src/handlers/actions/editDomain.ts:15]: I know this wasn't introduced here, but I want to mention that I like this pattern/refactor using the `transactionHasEvent` util

**Review by rdig — APPROVED** [review]
Looks all good. Nice work!

**Review by ArmandoGraterol — APPROVED** [review]
LGTM 💯

**Pull request merged**